stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/config.py

```diff
@@ -56,6 +56,10 @@ def _es_config() -> Dict[str, Any]:
     if (u := os.getenv("ES_USER")) and (p := os.getenv("ES_PASS")):
         config["http_auth"] = (u, p)
 
+    # Include timeout setting if set
+    if timeout := os.getenv("ES_TIMEOUT"):
+        config["timeout"] = timeout
```

Comment:
Hi. Nice addition. Should this be `request_timeout`? https://www.elastic.co/guide/en/elasticsearch/client/python-api/8.18/config.html#timeouts